lib/vanity/frameworks/rails.rb

```diff
@@ -43,11 +43,15 @@ module UseVanity
       def use_vanity(method_name = nil, &block)
         define_method(:vanity_identity_block) { block }
         define_method(:vanity_identity_method) { method_name }
-
-        around_filter :vanity_context_filter
-        before_filter :vanity_reload_filter unless ::Rails.configuration.cache_classes
-        before_filter :vanity_query_parameter_filter
-        after_filter :vanity_track_filter
+        unless respond_to?(:before_action)
+          alias_method :before_action, :before_filter
```

Comment:
Hm, this is still throwing `/home/travis/build/assaf/vanity/lib/vanity/frameworks/rails.rb:47:in 'alias_method': undefined method 'before_filter' for class 'AbTestController' (NameError)`. 

This is a bit ugly, but what do you think about:
```ruby
callback_method_name = respond_to?(:before_action) ? :action : :filter

send(:"around_#{callback_method_name}", :vanity_context_filter)
...
```